app/uk/gov/hmrc/cataloguefrontend/connector/TeamsAndRepositoriesConnector.scala

```diff
@@ -173,9 +173,6 @@ case class GitRepository(
 ) {
   def isShared: Boolean =
```

Comment:
`isShared` is only used by the partial - it could be moved into the partial, along with the `Constant.sharedRepoTeamsCutOff`.

Reply:
Actually, it looks like `prcommenter/model.scala` also duplicates this - can this also use the new partial?

Reply:
Updated `prcommenter/model.scala` and added tests for the PrCommenter page

Reply:
Can `isShared` be removed from here now?